storage/innobase/include/buf0buf.h

```diff
@@ -1603,15 +1602,13 @@ class buf_pool_t
 	ut_allocator<unsigned char>	allocator;	/*!< Allocator used for
 					allocating memory for the the "chunks"
 					member. */
-	volatile ulint	n_chunks;	/*!< number of buffer pool chunks */
-	volatile ulint	n_chunks_new;	/*!< new number of buffer pool chunks */
+	ulint		n_chunks;	/*!< number of buffer pool chunks */
+	ulint		n_chunks_new;	/*!< new number of buffer pool chunks */
```

Comment:
Comments should say what is protecting each data field. It seems to be `buf_pool.mutex`.